File: App/Messaging/Microservices.Demo.Messages/Services/Policies/Dtos/PersonDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Microservices.Demo.Messages.Services.Policies.Dtos
{
    public class PersonDto
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string TaxId { get; set; }
    }
}

File: App/Clients/ng-microservices-demo-client/src/app/infrastructure/rest-clients/policies/models/dtos/person-dto.ts
import { AutoMap } from "@automapper/classes";
export class PersonDto {
  @AutoMap()
  public firstName: string;
  @AutoMap()
  public lastName: string;
  @AutoMap()
  public taxId: string;

  constructor(firstName: string = '', lastName: string = '', taxId: string = '') {
    this.firstName = firstName;
    this.lastName = lastName;
    this.taxId = taxId;
  }
}
